zebra-chain/src/transparent.rs

```diff
@@ -1,7 +1,7 @@
 //! Transparent-related (Bitcoin-inherited) functionality.
 #![allow(clippy::unit_arg)]
 
-mod address;
+pub mod address;
```

Comment:
We probably dont want to do this however it was the only way i found by now to use the `to_address()` added in https://github.com/ZcashFoundation/zebra/pull/1191 in `founders_reward.rs`

Reply:
I think we want zebra-chain users to be able to use transparent addresses, so this change is good.